snapcraft/plugins/kernel.py

```diff
@@ -531,3 +531,8 @@ def do_install(self):
                 os.path.join(self.installdir, "lib", "firmware"), self.installdir
             )
         os.rmdir(os.path.join(self.installdir, "lib"))
+        # install .config as config-$version
+        config = "config-{}".format(self.kernel_release)
+        config_path = os.path.join(self.installdir, config)
```

Comment:
how about installing it under `(self.installdir, "snap", config)`? This way it sits together with every other manifesty thing.

Reply:
it sits next to the Symbols file with the above code and also matches where the config lives in the deb based kernel snaps, if we change it to go to the other dir, we should change the deb based snap builds alongside for consistency.